docs/docs/intro/configure.md

```diff
@@ -343,6 +344,42 @@ marathon {
 </TabItem>
 </Tabs>
 
+NOTE: analyticsTracking can also be enabled (default value) / disabled directly from the CLI. It is disabled if it's set to be disabled in either the config or the CLI.
```

Comment:
Can you please just use https://docusaurus.io/docs/markdown-features/admonitions?
There are examples in the codebase for them already